<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WPurgaV</class>
 <widget class="QWidget" name="WPurgaV">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>175</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>175</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>175</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="lbVbvBT">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>78</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>VBVB</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="WLabel" name="lbVbvsel">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>46</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="id">
    <string>p_Fix32_THISNODE_VBVSEL_F_CV</string>
   </property>
   <property name="Fondo_Inicial" stdset="0">
    <enum>WLabel::Sin_color</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="Alineacion" stdset="0">
    <enum>WLabel::Centro</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbVbvaU">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>62</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="WLabel" name="lbVbvB">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>78</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="id">
    <string>p_Fix32_THISNODE_VBVB_F_CV</string>
   </property>
   <property name="Fondo_Inicial" stdset="0">
    <enum>WLabel::Sin_color</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="Alineacion" stdset="0">
    <enum>WLabel::Centro</enum>
   </property>
  </widget>
  <widget class="WLabel" name="lbVbvMa">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>110</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="Fondo_Inicial" stdset="0">
    <enum>WLabel::Sin_color</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="Alineacion" stdset="0">
    <enum>WLabel::Centro</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbVbvdmdU">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>30</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="WLabel" name="lbVbvOhms">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>126</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="Fondo_Inicial" stdset="0">
    <enum>WLabel::Sin_color</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="Alineacion" stdset="0">
    <enum>WLabel::Centro</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbVbvBU">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>78</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbVbvOhmsU">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>126</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>OHMS</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbVbvselT">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>46</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>VBVSEL</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbVbvselU">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>46</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbVbvMaT">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>VBVMA</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbVbvOhmsT">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>126</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>VBVOHMS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbVbvdmdT">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>30</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>VBVDMD</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="WLabel" name="lbVbvdmd">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>30</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="id">
    <string>p_Fix32_THISNODE_VBVDMD_F_CV</string>
   </property>
   <property name="Fondo_Inicial" stdset="0">
    <enum>WLabel::Sin_color</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="Alineacion" stdset="0">
    <enum>WLabel::Centro</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbVbvaT">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>62</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>VBVA</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="WLabel" name="lbVbva">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>62</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="id">
    <string>p_Fix32_THISNODE_VBVA_F_CV</string>
   </property>
   <property name="Fondo_Inicial" stdset="0">
    <enum>WLabel::Sin_color</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="Alineacion" stdset="0">
    <enum>WLabel::Centro</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbVbvMaU">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>110</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>MA</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbFondo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>175</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(128, 128, 128);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btCerrar">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>173</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(192, 192, 192);</string>
   </property>
   <property name="text">
    <string>PURGA VARIABLE</string>
   </property>
  </widget>
  <zorder>lbFondo</zorder>
  <zorder>lbVbvBT</zorder>
  <zorder>lbVbvsel</zorder>
  <zorder>lbVbvaU</zorder>
  <zorder>lbVbvB</zorder>
  <zorder>lbVbvMa</zorder>
  <zorder>lbVbvdmdU</zorder>
  <zorder>lbVbvOhms</zorder>
  <zorder>lbVbvBU</zorder>
  <zorder>lbVbvOhmsU</zorder>
  <zorder>lbVbvselT</zorder>
  <zorder>lbVbvselU</zorder>
  <zorder>lbVbvMaT</zorder>
  <zorder>lbVbvOhmsT</zorder>
  <zorder>lbVbvdmdT</zorder>
  <zorder>lbVbvdmd</zorder>
  <zorder>lbVbvaT</zorder>
  <zorder>lbVbva</zorder>
  <zorder>lbVbvMaU</zorder>
  <zorder>btCerrar</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WLabel</class>
   <extends>QWidget</extends>
   <header>wlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
